src/main/java/org/kohsuke/github/GHRepository.java

```diff
@@ -866,26 +849,62 @@ public GHCommitStatus getLastCommitStatus(String sha1) throws IOException {
     }
 
```

Comment:
Lines 851 - 889 are the primary additions to the class: the methods `getLicense`, `getFullLicense` and `getLicenseContent`